tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -209,7 +210,10 @@ public static function dataFile(): iterable
 		$base = dirname(__DIR__, 3) . '/';
```

Comment:
using `DIRECTORY_SEPARATOR` and `dirname` in the `findTestFiles` should be enough and faster

Reply:
it would not resolve `..` to real path as described in https://github.com/phpstan/phpstan-src/pull/3449#issuecomment-2354702992

Reply:
The 2nd part of my answer would ;-)